content/faq/how-rotate-kubernetes-ssh-keys.md

```diff
@@ -0,0 +1,72 @@
+-=----
+title: "How to rotate ssh keys for Kubernetes?"
+description: "Learn how to rotate ssh keys for kubernetes."
```

Comment:
"Learn how to rotate ssh keys on a kops cluster by generating new ones and then performing a rolling-update on the cluster to apply the changes."